app/(main)/album/[id]/loading.tsx

```
@@ -0,0 +1,28 @@
+import Skeleton from "@/app/components/loading/skeleton";
+
+export default function Loading() {
+  return (
+    <div className="container mx-auto px-4 py-8 animate-pulse">
+    <Skeleton className="h-8 w-1/4 mb-4"></Skeleton>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)